webapp/src/index.tsx

```diff
@@ -75,8 +75,23 @@ export default class Plugin {
 
         let rhs: any = null;
         if (isRHSCompatable()) {
-            rhs = registry.registerRightHandSidebarComponent(RHS, RHSTitle);
-            setOpenRHSAction(rhs.showRHSPlugin);
+            if (!registry.registerAppBarComponent) {
```

Comment:
We probably should fall back to the old method instead of returning here. (Unless they are on the same level of compatibility then we can just check in isRHSCompatable)

Side note: I really need to get back in to the code here and figure out what we actually need to support for backwards compatibility here.